frontend/src/ScoreBox.js

```diff
@@ -265,14 +265,14 @@ class ScoreBox extends Component {
             if (
               ENABLE_RELATIVE_COLORS &&
               this.state.selected in infoMap &&
-              props.original.key !== this.state.selected
+              key !== this.state.selected
             ) {
               const other =
                 infoMap[this.state.selected][
                   this.state.isAverage ? 'average_reviews' : 'recent_reviews'
-                ][props.column.id]
-              if (Math.abs(value - other) > 0.01) {
-                if (value > other) {
+                ][column.id]
```

Comment:
can you destructure this out of column?

Reply:
I think it makes sense here since there's no other values being used in column, iirc